jcl/src/java.management/share/classes/java/lang/management/ThreadMXBean.java

```diff
@@ -633,5 +633,105 @@
 	 */
 	public ThreadInfo[] dumpAllThreads(boolean lockedMonitors,
 			boolean lockedSynchronizers);
+	/*[IF Java10]*/
 
+	/**
+	 * Returns an array of {@link ThreadInfo} objects holding information on all
+	 * threads that were alive when the call was invoked.
+	 * 
+	 * @param lockedMonitors
+	 *            boolean indication of whether or not information on all
+	 *            currently locked object monitors is to be included in the
+	 *            returned array
+	 * @param lockedSynchronizers
+	 *            boolean indication of whether or not information on all
+	 *            currently locked ownable synchronizers is to be included in
+	 *            the returned array
+	 * @param maxDepth limits the number of stack frames returned
+	 * @return an array of <code>ThreadInfo</code> objects
+	 * @throws SecurityException
+	 *             if there is a security manager in effect and the caller does
+	 *             not have {@link ManagementPermission} of &quot;monitor&quot;.
+	 * @throws UnsupportedOperationException
+	 *             if either of the following conditions apply:
+	 *             <ul>
+	 *                 <li><code>lockedMonitors</code> is <code>true</code> but
+	 *                     a call to {@link #isObjectMonitorUsageSupported()} would
+	 *                     result in a <code>false</code> value
+	 *                 <li><code>lockedSynchronizers</code> is <code>true</code>
+	 *                     but a call to {@link #isSynchronizerUsageSupported()} would
+	 *                     result in a <code>false</code> value
+	 *             </ul>
+	 * 	@since 10
```

Comment:
There is some unnecessary indenting here and in the next method as well.

Reply:
That was copied that from the other versions.  I cleaned it up and added missing </li>s.
Do you want me to clean up the originals?

Reply:
I was referring to the extra tab before `@since 10`, which is still there.

Reply:
Fixed.  I will push the changes with changes from @llxia 's review.